main.go

```diff
@@ -97,13 +97,11 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "IngressNodeFirewall")
 		os.Exit(1)
 	}
-	/* FIXME: enable webhook
+
```

Comment:
we need to check here if webhook is enabled then we setup to allow running w/o webhook

Reply:
Done